Sources/Request.swift

```diff
@@ -4,14 +4,17 @@ public struct Request: Message {
     public var version: Version
     public var headers: Headers
     public var body: Body
-    public var storage: [String: Any] = [:]
+    public var cookies: [String: String]
```

Comment:
> For this middleware to compile it needs to depend on Vapor to use Vapor's cookies property. So it would have to depend on S4 and on Vapor.

Exactly. That's what I want to prevent by making `cookies` a part of `S4`. Middleware that uses cookies should not have to depend on a particular framework. Cookies are fundamental to HTTP.